docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -31,6 +31,28 @@ To send a message, you would run a command like:
 
 where ``myio`` is the name of your component.
 
+If you need to use extra information from your front end in your custom
+actions, you can add this information in the ``metadata`` dict of your user
+message. This information will accompany the user message through the rasa
+server into the action server when applicable, where you can find it stored in
+the ``tracker``. Message metadata will not directly affect NLU classification
+or action prediction.
+
+To send a message with metadata, you would run a command like:
```

Comment:
the thing is, that this is completely arbitrary. It depends on the input channel what is handled as metadata. Hence, I'd actually exclude this example (@erohmensing what do you think?)

Reply:
Exactly, if the `input_channl` doesn't handle the `metadata` this `curl` requisition will not work. Maybe it will be better to add this part in further PRs :thinking: Or improve the example at the end of the page

Reply:
We decided to remove this part of the documentation